packages/flame_test/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ version: 1.2.0
 homepage: https://github.com/flame-engine/flame/tree/main/packages/flame_test
 
 environment:
-  sdk: ">=2.14.0 <3.0.0"
+  sdk: ">=2.15.0 <3.0.0"
```

Comment:
Is this constraint upgrade needed for something? Because flame itself still has the `2.14` constraint and if for some reason someone is on `2.14` they can no longer test their games easily.

Reply:
Yep, so we can use constructors tear off, should we increase on flame too (on a separate PR probably)?

Reply:
Why not raise to 2.16 while we're at it? 
An important caveat though is that raising the SDK version is a **big** breaking change, so at the very least it has to be in a standalone PR so that it would be prominently visible in the changelog.

Reply:
Agreed, would keep it at 2.14 for now and then open a PR with the upgrade changes where we can discuss them more in depth.